diff -urNp -x '*.orig' libaal-1.0.7.org/configure.ac libaal-1.0.7/configure.ac
--- libaal-1.0.7.org/configure.ac	2017-11-23 12:27:22.000000000 +0100
+++ libaal-1.0.7/configure.ac	2026-03-03 00:10:54.165455515 +0100
@@ -127,7 +127,6 @@ else
   ALIGN_FLAGS="-falign-jumps=1 -falign-loops=1 -falign-functions=1"
 fi
 
-CFLAGS=""
 MINIMAL_CFLAGS=""
 GENERIC_CFLAGS=""
 
@@ -162,13 +161,9 @@ fi
 
 CFLAGS="$CFLAGS -W -Wall"
 
-if test x$enable_debug = xyes; then
-    GENERIC_CFLAGS="$GENERIC_CFLAGS -O0 -g"
-    MINIMAL_CFLAGS="$MINIMAL_CFLAGS -O0 -g"
-else
-    GENERIC_CFLAGS="$GENERIC_CFLAGS -O3"
-    MINIMAL_CFLAGS="$MINIMAL_CFLAGS -Os"
-
+GENERIC_CFLAGS="$CFLAGS"
+MINIMAL_CFLAGS="$CFLAGS -Os"
+if test x$enable_debug != xyes; then
     if test "x$uninitialized" = xyes; then
        CFLAGS="$CFLAGS -Wuninitialized"
     fi
