diff -burN libaal-0.5.3.orig/configure.in libaal-0.5.3/configure.in
--- libaal-0.5.3.orig/configure.in	2004-08-08 14:23:34.681272768 +0200
+++ libaal-0.5.3/configure.in	2004-08-08 14:25:00.416239072 +0200
@@ -126,7 +126,6 @@
   ALIGN_FLAGS="-malign-jumps=1 -malign-loops=1 -malign-functions=1 -malign-labels=1 -momit-frame-pointer -mno-inline -mstrength-reduce"
 fi
 
-CFLAGS=""
 ALONE_CFLAGS=""
 GENERIC_CFLAGS=""
 
@@ -158,13 +157,7 @@
 
 CFLAGS="$CFLAGS -W -Wall"
 
-if test x$enable_debug = xyes; then
-    GENERIC_CFLAGS="$GENERIC_CFLAGS -O0 -g"
-    ALONE_CFLAGS="$ALONE_CFLAGS -O0 -g"
-else
-    GENERIC_CFLAGS="$GENERIC_CFLAGS -O3"
-    ALONE_CFLAGS="$ALONE_CFLAGS -Os"
-
+if test x$enable_debug != xyes; then
     if test "x$uninitialized" = xyes; then
        CFLAGS="$CFLAGS -Wuninitialized"
     fi
