--- libaal-1.0.2/configure.ac.orig	2004-10-21 20:00:15.000000000 +0200
+++ libaal-1.0.2/configure.ac	2004-10-24 21:50:57.560925672 +0200
@@ -126,7 +126,6 @@
   ALIGN_FLAGS="-malign-jumps=1 -malign-loops=1 -malign-functions=1"
 fi
 
-CFLAGS=""
 MINIMAL_CFLAGS=""
 GENERIC_CFLAGS=""
 
@@ -158,13 +157,9 @@
 
 CFLAGS="$CFLAGS -W -Wall"
 
-if test x$enable_debug = xyes; then
-    GENERIC_CFLAGS="$GENERIC_CFLAGS -O0 -g"
-    MINIMAL_CFLAGS="$MINIMAL_CFLAGS -O0 -g"
-else
-    GENERIC_CFLAGS="$GENERIC_CFLAGS -O3"
-    MINIMAL_CFLAGS="$MINIMAL_CFLAGS -Os"
-
+GENERIC_CFLAGS="$CFLAGS"
+MINIMAL_CFLAGS="$CFLAGS -Os"
+if test x$enable_debug != xyes; then
     if test "x$uninitialized" = xyes; then
        CFLAGS="$CFLAGS -Wuninitialized"
     fi
